component/outbound/filter.go

```diff
@@ -72,7 +73,12 @@ func (s *DialerSet) filterHit(dialer *dialer.Dialer, filters []*config_parser.Fu
 			for _, param := range filter.Params {
 				switch param.Key {
 				case FilterKey_Name_Regex:
-					matched, _ := regexp.MatchString(param.Val, dialer.Property().Name)
+					regex, err := regexp2.Compile(param.Val, 0)
+					if err != nil {
+						logrus.Warnln(err, ",No filtering")
```

Comment:
Why this?

Reply:
Does it mean to use logrus directly here, or what?